xcode/Subconscious/Shared/Parsers/SubtextAttributedStringRenderer.swift

```diff
@@ -86,6 +86,12 @@ extension URL {
     }
 }
 
+extension SubSlashlinkLink {
+    func toEntryLink() -> EntryLink {
+        EntryLink(address: slashlink, title: text)
+    }
+}
```

Comment:
Conversion function. We parse SubSlashlinkLink from URL and then immediately convert it to an EntryLink in most cases.